lib/phoenix_live_reload/live_reloader.ex

```diff
@@ -141,13 +140,12 @@ defmodule Phoenix.LiveReloader do
     end
   end
 
-  defp has_body?(resp_body), do: String.contains?(resp_body, "<body")
+  defp has_body?(resp_body), do: :string.find(resp_body, "<body") != :nomatch
```

Comment:
We work with old Elixir versions, we cannot assume everyone is on latest Erlang/OTP. Also, this change offers no benefit AFAIK?

```suggestion
  defp has_body?(resp_body), do: String.contains?(resp_body, "<body")
```

Reply:
`String.contains?/2` require binary subject, while `:string.find/2` can work with `:unicode.chardata/0` which mean that `["<bo", 'dy']` will work with second approach.